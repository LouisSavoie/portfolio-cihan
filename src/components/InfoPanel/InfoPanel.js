import React from 'react'
import Card from 'react-bootstrap/Card'
import Button from 'react-bootstrap/Button'

const Info = () => {
  return (
    <Card data-testid="info-panel" style={{ width: '18rem' }}>
      <Card.Img variant="top" src="holder.js/100px180" />
      <Card.Body>
        <Button variant="primary" size="lg">Button</Button>
        <Button variant="primary" size="lg">Button</Button>
        <Button variant="primary" size="lg">Button</Button>
        <Button variant="primary" size="lg">Button</Button>
      </Card.Body>
    </Card>
  )
}

export default Info