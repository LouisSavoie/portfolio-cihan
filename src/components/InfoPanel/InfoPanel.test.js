import React from 'react'
import { render, screen, cleanup } from '@testing-library/react'
import '@testing-library/jest-dom'
import renderer from 'react-test-renderer'
import InfoPanel from './InfoPanel'

afterEach(() => {
  cleanup()
})

test('should render InfoPanel', () => {
  render(<InfoPanel />)
  const InfoPanelElement = screen.getByTestId('info-panel')
  expect(InfoPanelElement).toBeInTheDocument()
})

test('matches snapshot', () => {
  const tree = renderer.create(<InfoPanel />).toJSON()
  expect(tree).toMatchSnapshot()
})
