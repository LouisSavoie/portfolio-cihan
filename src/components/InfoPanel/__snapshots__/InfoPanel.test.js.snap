// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="container"
  data-testid="info-panel"
>
  <div
    className="card"
  >
    <div
      className="justify-content-center row row-cols-md-2 row-cols-sm-1 row-cols-1"
    >
      <div
        className="pe-md-0 col-xxl-4 col"
      >
        <img
          className="card-img"
          src="./img/photo.png"
          style={
            Object {
              "borderRadius": "1rem",
            }
          }
        />
        <div
          className="d-none d-md-flex d-lg-none justify-content-center flex-wrap card-body"
        >
          <a
            className="btn btn-primary"
            disabled={false}
            href="https://baconipsum.com/"
            onClick={[Function]}
            onKeyDown={[Function]}
            rel="noopener noreferrer"
            role="button"
            tabIndex={0}
            target="_blank"
          >
            Resume
          </a>
          <a
            className="btn btn-primary"
            disabled={false}
            href="mailto:thelousavoie@gmail.com"
            onClick={[Function]}
            onKeyDown={[Function]}
            role="button"
            tabIndex={0}
          >
            Email
          </a>
          <a
            className="btn btn-primary"
            disabled={false}
            href="https://www.linkedin.com/in/louis-savoie-6a70811b4/"
            onClick={[Function]}
            onKeyDown={[Function]}
            rel="noopener noreferrer"
            role="button"
            tabIndex={0}
            target="_blank"
          >
            LinkedIn
          </a>
          <a
            className="btn btn-primary"
            disabled={false}
            href="https://github.com/LouisSavoie"
            onClick={[Function]}
            onKeyDown={[Function]}
            rel="noopener noreferrer"
            role="button"
            tabIndex={0}
            target="_blank"
          >
            GitHub
          </a>
          <a
            className="btn btn-primary"
            disabled={false}
            href="https://www.leetcode.com/lousavoie"
            onClick={[Function]}
            onKeyDown={[Function]}
            rel="noopener noreferrer"
            role="button"
            tabIndex={0}
            target="_blank"
          >
            LeetCode
          </a>
          <a
            className="btn btn-primary"
            disabled={false}
            href="https://www.codewars.com/users/LouisSavoie"
            onClick={[Function]}
            onKeyDown={[Function]}
            rel="noopener noreferrer"
            role="button"
            tabIndex={0}
            target="_blank"
          >
            Codewars
          </a>
        </div>
      </div>
      <div
        className="ps-md-0 col-xxl-8 col"
      >
        <div
          className="card-body"
        >
          <h1
            className="text-center card-title"
          >
            Bacon Tester Ipsum
          </h1>
          <h3
            className="text-center card-title"
          />
          <div
            className="d-xs-flex d-sm-flex d-md-none d-lg-flex justify-content-center flex-wrap"
          >
            <a
              className="btn btn-primary"
              disabled={false}
              href="https://baconipsum.com/"
              onClick={[Function]}
              onKeyDown={[Function]}
              rel="noopener noreferrer"
              role="button"
              tabIndex={0}
              target="_blank"
            >
              Resume
            </a>
            <a
              className="btn btn-primary"
              disabled={false}
              href="mailto:thelousavoie@gmail.com"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
              tabIndex={0}
            >
              Email
            </a>
            <a
              className="btn btn-primary"
              disabled={false}
              href="https://www.linkedin.com/in/louis-savoie-6a70811b4/"
              onClick={[Function]}
              onKeyDown={[Function]}
              rel="noopener noreferrer"
              role="button"
              tabIndex={0}
              target="_blank"
            >
              LinkedIn
            </a>
            <a
              className="btn btn-primary"
              disabled={false}
              href="https://github.com/LouisSavoie"
              onClick={[Function]}
              onKeyDown={[Function]}
              rel="noopener noreferrer"
              role="button"
              tabIndex={0}
              target="_blank"
            >
              GitHub
            </a>
            <a
              className="btn btn-primary"
              disabled={false}
              href="https://www.leetcode.com/lousavoie"
              onClick={[Function]}
              onKeyDown={[Function]}
              rel="noopener noreferrer"
              role="button"
              tabIndex={0}
              target="_blank"
            >
              LeetCode
            </a>
            <a
              className="btn btn-primary"
              disabled={false}
              href="https://www.codewars.com/users/LouisSavoie"
              onClick={[Function]}
              onKeyDown={[Function]}
              rel="noopener noreferrer"
              role="button"
              tabIndex={0}
              target="_blank"
            >
              Codewars
            </a>
          </div>
          <p
            className="card-text"
          >
            Prosciutto short loin filet mignon boudin, doner leberkas jerky bresaola sirloin tenderloin. Ham landjaeger doner, fatback tri-tip pork loin rump kevin venison. Jowl kielbasa frankfurter doner alcatra tenderloin, porchetta bresaola hamburger tail picanha meatball chislic shank. Burgdoggen meatloaf short ribs beef, shank pork pig fatback andouille meatball brisket pancetta strip steak. Burgdoggen shankle frankfurter pork loin tail beef kevin shoulder bacon landjaeger ham corned beef spare ribs.
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
